### DISCUSSION TITLE
Devops-practive ami is not available in emea region

### DISCUSSION THREAD

**Discussion by asrajannn** [learndevopsonline/learndevopsonline#1983] [Q&A]
Hi,

I want to tryout whatever is explained in the video. I tried to spin up a ec2 instance with devops-practice ami, i didn't find any ami with the given name.

Tried changing the location to N.virginia, still couldn't get the ami.

please suggest an alernative.
![WhatsApp Image 2024-10-06 at 1 53 25 PM](https://github.com/user-attachments/assets/bb309877-0859-414c-8ca6-d8de876a35d3)

**Comment by chaithanya1812** [comment]
Hi @asrajannn  Select this AMI.
![image](https://github.com/user-attachments/assets/8c33a583-9dd7-4391-83c4-e54dda749862)

**Reply by hasifasi** [reply]
Devops- practice is a search key word.. it returns you 2 AMI with 2 different OS, one is Centos and another one is Red Hat... we are using Red Hat
